src/java.base/share/native/libzip/zip_util.c

```diff
@@ -764,7 +764,7 @@ readCEN(jzfile *zip, jint knownTotal)
  * Opens a zip file with the specified mode. Returns the jzfile object
  * or NULL if an error occurred. If a zip error occurred then *pmsg will
  * be set to the error message text if pmsg != 0. Otherwise, *pmsg will be
- * set to NULL. Caller is responsible to free the error message.
+ * set to NULL. Caller doesn't need to free the error message.
```

Comment:
I'd put some more context here why the caller does not need to free. (as it is a static text)

Reply:
Added; also mentioned that we want the buffer to be thread-safe. Let me know if that's what you had in mind.